docs.css

```diff
@@ -0,0 +1,87 @@
+.logo--header {
+  width: 4rem;
+}
+
+header {
+  padding: 1rem;
+  display: flex;
+  column-gap: 20px;
+  align-items: center;
+  grid-area: header;
+  box-shadow: var(--green-teal) 2px 5px 5px;
+}
+
+body {
+  background-color: var(--background-color);
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  grid-template-areas: "header header" "aside main";
+  grid-template-rows: 5rem 1fr;
+  grid-template-columns: 15%;
+  box-sizing: border-box;
+}
+
+main {
+  grid-area: main;
+  overflow-y: scroll;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 30px 50px;
+}
+
+aside {
```

Comment:
Again don't grab elements.

Reply:
Is it okay if I'm using only in docs page? Because I haven't used it in any of the components.

Reply:
ok